tests/TestUtil.php

```diff
@@ -45,6 +45,7 @@ public static function getConnection(): PDO
             $dsn .= 'dbname=' . $connectionParams['dbname'] . $separator;
             $dsn = rtrim($dsn);
             self::$connection = new PDO($dsn, $connectionParams['user'], $connectionParams['password']);
+            self::$connection->query("SET NAMES '".$connectionParams['charset']."'");
```

Comment:
I have not found a connection option which would work for postgre. Thats the way how doctrine is bypass this issue.